test/NuGet.CommandLine.Test/CommandRunner.cs

```diff
@@ -12,7 +12,7 @@ public class CommandRunner
     {
         // Item1 of the returned tuple is the exit code. Item2 is the standard output, and Item3 
         // is the error output.
-        public static Tuple<int, string, string> Run(string process, string workingDirectory, string arguments, bool waitForExit, int timeOutInMilliseconds = 60000,
+        public static Tuple<int, string, string> Run(string process, string workingDirectory, string arguments, bool waitForExit, int timeOutInMilliseconds = 600000000,
```

Comment:
good catch, I change the timeout for debugging purposes, will revert this change